openid_connect_configuration.conf

```diff
@@ -35,7 +43,54 @@ map $host $oidc_scopes {
 map $host $oidc_logout_redirect {
     # Where to send browser after requesting /logout location. This can be
     # replaced with a custom logout page, or complete URL.
-    default "/_logout"; # Built-in, simple logout page
+
+    default "/_logout"; # Built-in, simple logout page for NGINX R28 
+}
+
+map $host $oidc_logout_redirect_uri { # use for NGINX R29+ versions
+    # This is the redirect URI which is called by IdP to erase cookies and
+    # redirect to the original page or custom logout page after successfully
+    # ending OIDC session from IdP.
+    default "/v2/_logout";
+}
+
+map $host $post_logout_return_uri {
+    # The following examples can be replaced with a custom logout page, or
+    # a complete URL to be redirected after successful logout from the IdP.
+
+    # Example 1: Redirect to the original langding page.
+    #            ./docker/build-context/nginx/sample/proxy_server_frontend.conf
+    #              -> redirect to the '/' location block
+    #            ./docker/build-context/content/index.html
+    #
+    default $redirect_base;
+
+    # Example 2: Redirect to a custom logout page
+    #            ./docker/build-context/nginx/sample/proxy_server_frontend.conf
+    #              -> redirect to the '/signout' location block
+    #            ./docker/build-context/content/signout.html
+    #
+    # default $redirect_base/signout;
+
+    # Example 3: Redirect to an another URL
+    # default https://www.nginx.com;
+}
+
+map $host $return_token_to_client_on_login {
```

Comment:
Since the `ID Token` is stored in a variable, it can also be returned from the app-specific config. Not sure if we need this option.